lisa/sut_orchestrator/azure/features.py

```diff
@@ -1905,3 +1905,77 @@ def can_disable(cls) -> bool:
 
     def enabled(self) -> bool:
         return True
+
+
+@dataclass_json()
+@dataclass()
+class ArchitectureSettings(schema.FeatureSettings):
+    type: str = "Architecture"
+    # Architecture in hyper-v
+    arch: str = "x64"
```

Comment:
restrict it to accept one of x64 and arm64.

Reply:
updated